index.html

```diff
@@ -240,6 +240,9 @@ <h2 class="display-4">
 				</a>
 				was adapted for What is a Book in the 21st Century? Working with Historical Texts in a Digital Environment (History GR8975) at Columbia University. This graduate-level course introduces students to techniques of working in digital environments as they work collectively to create a small scale digital edition of a 16th-century manuscript. Completing the framework was a collaborative effort between the Center for Teaching and Learning and the course team. 
 				</p>
+				<p>
+					See the DLCC for GR8975 <a href="https://jebrodsky.github.io/diglit-gr8975/#/all/" target="_blank">here</a>.
```

Comment:
This should be relative not absolute - change to `/diglit-gr8975/#/all/` so it's not referencing your repository